### PR TITLE
Kubernetes CDK stack

### PR DESCRIPTION
**Pull request by ajvpot**
Creates a CDK stack that deploys a kube cluster in the new account.

### CODE REVIEW

**Review by freeqaz — CHANGES_REQUESTED** [review]
I don't think there is any reason to make a new folder for this. We should just add this as a new "Stack" to the existing CDK folder. We can independently deploy this EKS stuff from the existing stuff too. (Just make a new JSON file for stack-inputs instead of a new folder?)

**Comment by ajvpot** [comment]
> I don't think there is any reason to make a new folder for this. We should just add this as a new "Stack" to the existing CDK folder. We can independently deploy this EKS stuff from the existing stuff too. (Just make a new JSON file for stack-inputs instead of a new folder?)

I wanted the new package versions, we should be good to merge this folder into the other one after you land your updates.

**Pull request closed without merging**
